meet-data/ipf/ipf-standardize-csv

```diff
@@ -124,11 +124,15 @@ def unreverse_names(csv):
 
         row[nameidx] = name
 
-
+def hunt_dq(csv):
+    plidx = csv.index('Place')
+    for row in csv.rows:
+        if row[plidx] == '':
+            row[plidx] = 'DQ'
 def main(filename):
     csv = Csv()
-    with open(filename, 'r') as fd:
-        csv.rows = [x.strip().split(',') for x in fd.readlines()]
+    with codecs.open(filename, 'r', encoding="utf-8",  errors='ignore') as fd:
+        csv.rows = [x.strip().replace("?","").split(',') for x in fd.readlines()]
```

Comment:
Do not silently remove errors. The "?" are important, because they mark places that you will have to fix manually. This is particularly important for names -- we can't just silently drop random letters because of encoding errors.

Reply:
Problem is I am getting a lot more "?" than you. without this line, the script actually fails.

Reply:
Is there a way to remove the "?" more selectively, so that names don't get mangled?